azure-pipelines/main-pipeline.yml

```diff
@@ -7,6 +7,7 @@ trigger:
       - main
       - staging
       - development
+      - feature_cd
```

Comment:
i assume this was for testing, so this can be taken out, no?

Reply:
WIll be removed in next PR.